From d3e2f335036aa4f899f01d4dd0199685893a1837 Mon Sep 17 00:00:00 2001
From: Dan Streetman <ddstreet@ieee.org>
Date: Mon, 21 Aug 2023 19:25:49 -0400
Subject: [PATCH] cryptenroll: change man page example to remove leading 0x and
 lowercase hex

(cherry picked from commit a11a2e059cd8e7d488985c23ceee0d6a2d5c8fb0)

Related: RHEL-16182
---
 man/systemd-cryptenroll.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd-cryptenroll.xml b/man/systemd-cryptenroll.xml
index 3f36f320b6..28e315bd1c 100644
--- a/man/systemd-cryptenroll.xml
+++ b/man/systemd-cryptenroll.xml
@@ -391,9 +391,9 @@
         specifies that PCR registers 4, 1, and 5 should be used.</para>
         <para>Example: <option>--tpm2-pcrs=7:sha256</option> specifies that PCR register 7 from the SHA256
         bank should be used.</para>
-        <para>Example: <option>--tpm2-pcrs=4:sha1=0x3A3F780F11A4B49969FCAA80CD6E3957C33B2275</option>
+        <para>Example: <option>--tpm2-pcrs=4:sha1=3a3f780f11a4b49969fcaa80cd6e3957c33b2275</option>
         specifies that PCR register 4 from the SHA1 bank should be used, and a hash digest value of
-        0x3A3F780F11A4B49969FCAA80CD6E3957C33B2275 will be used instead of reading the current PCR
+        3a3f780f11a4b49969fcaa80cd6e3957c33b2275 will be used instead of reading the current PCR
         value.</para>
         </listitem>
       </varlistentry>
